### ISSUE TITLE
Resource paths don't exist.

### ISSUE THREAD

**Issue by Robokan** [isaac-sim/IsaacLab#446]
### Describe the bug

All of the examples I have tried fail  because the resources don't exist. Try any example from the "Running Existing Scripts".

### Steps to reproduce

./isaaclab.sh -p source/standalone/demos/quadrupeds.py
or
./isaaclab.sh -p source/standalone/workflows/rsl_rl/train.py --task Isaac-Reach-Franka-v0 --headless

```
FileNotFoundError: USD file not found at path: 'http://omniverse-content-production.s3-us-west-2.amazonaws.com/Assets/Isaac/2023.1.1/Isaac/IsaacLab/Robots/ANYbotics/ANYmal-B/anymal_b.usd'
```

### System Info

Describe the characteristic of your environment:

- Isaac Sim Version: 2023.1.1
- OS: Ubuntu 20.04
- GPU: RTX 4090
- CUDA: 12.2
- GPU Driver: 535.54.03

### Checklist

- [x] I have checked that there is no similar issue in the repo (**required**)
- [x] I have checked that the issue is not in running Isaac Sim itself and is related to the repo

### Acceptance Criteria

- [ ] Scripts should run correctly even with 2023.1.1.

**Comment by Mayankm96** [comment]
Good catch. Can you replace this to: `f"{ISAAC_NUCLEUS_DIR}/Orbit"` for now? In Isaac Sim 2023.1.1, the assets are still available under "Orbit" directory on the Nucleus.

> https://github.com/isaac-sim/IsaacLab/blob/main/source/extensions/omni.isaac.lab/omni/isaac/lab/utils/assets.py#L54-L55

**Comment by JunghwanRo** [comment]
Were you able to run IsaacLab with 2023.1.1?

**Comment by KevinGeng3** [comment]
same question

**Comment by 0123gabriel** [comment]
I have the same question, but I cannot load this asset: Robots/Clearpath/Jackal/jackal.usd, is it still available? I can find it in isaacsim, but when I put that directory, and launch my train, it says that the directory doesn't exists.

**Comment by maxkra15** [comment]
Thanks @Robokan for reporting the missing asset paths.

This report targets Isaac Sim 2023.1.1 and its retired Nucleus/Orbit asset layout. Isaac Lab now supports Isaac Sim 6.1; Isaac Sim 5.1 and older are unsupported. We are closing this obsolete compatibility report. Please reopen with the exact URI and versions if a missing asset path still reproduces on a current Isaac Lab and Isaac Sim 6.1 installation.
